pom.xml

```diff
@@ -94,10 +94,11 @@
         <org.slf4j.version>1.7.12</org.slf4j.version>
         <jetty.version>9.4.19.v20190610</jetty.version>
         <jax.rs.api.version>2.1</jax.rs.api.version>
-        <jersey.version>2.27</jersey.version>
+        <jersey.server.version>2.29.1</jersey.server.version>
+        <jersey.client.version>2.27</jersey.client.version>
         <jackson.version>2.9.9</jackson.version>
         <jackson.databind.version>2.9.10.3</jackson.databind.version>
```

Comment:
looks like jackson can go to 2.10.3. also not sure why we are at separate versions for jackson and jackson databank

Reply:
Updated to 2.10.3 and removed the jackson.databind.version parameter and changed all dependencies to use only jackson.version.